WordPress/Classes/ViewRelated/Post/AbstractPostListViewController.m

```diff
@@ -697,6 +699,28 @@ - (CGFloat)heightForSearchWrapperView
 
 #pragma mark - Filter related
 
+- (BOOL)canFilterByAuthor
+{
+    return [self.blog isMultiAuthor] && self.blog.account.userID && [self.blog isHostedAtWPcom];
+}
+
+- (BOOL)shouldShowOnlyMyPosts
+{
+    PostAuthorFilter filter = [self currentPostAuthorFilter];
+    return filter == PostAuthorFilterMine;
+}
+
+- (PostAuthorFilter)currentPostAuthorFilter
+{
+    return PostAuthorFilterEveryone;
+}
+
+- (void)setCurrentPostAuthorFilter:(PostAuthorFilter)filter
+{
+    // Noop. The default implementation is read only.
+    // Subclasses may override the getter and setter for their own filter storage.
```

Comment:
Maybe this method doesn't need to be defined here and in `AbstractPostListViewControllerSubclass.h`. Since `PostListViewController` is the only class that implements and calls it, it could be just a regular private method in that class